src/compileModel.ts

```diff
@@ -124,10 +124,10 @@ const compileModel = <TSchema extends GenericObject = GenericObject>(
 		): Promise<Model | null> {
 			const { userDefined } = options;
 
-			const data = await Model.connection.executeDbFeature(
+			const data = await this.connection.executeDbFeature(
```

Comment:
Are these interchangeable in JS?

Reply:
In a static member, `this` references the other static members, so using the class name isn't necessary.  I'm not sure if this wasn't always the case or if I misunderstood it until recently: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static#calling_static_members_from_another_static_method